### PR TITLE
Update for 1.15 release

### PR DESCRIPTION
**Pull request by coverbeck**
SEAB-6086

### CODE REVIEW

File: news.html [hunk 1]
```diff
@@ -3,44 +3,44 @@
...
-             href="https://github.com/dockstore/dockstore/releases/tag/1.14.0">
-              1.14 release</a>
+             href="https://github.com/dockstore/dockstore/releases/tag/1.15.0">
+              1.15 release</a>
       </h5>
-        <span>Get more information on our 1.14.x series releases here.</span>
+        <span>Get more information on our 1.15.x series releases here.</span>
     </div>
-    <span class="date-display ml-3">May 31, 2023</span>
+    <span class="date-display ml-3">Jan 31, 2024</span>
```
Comment (coverbeck) [inline news.html:11]: Date is a placeholder, will update with actual release

File: news.html [hunk 2]
```diff
@@ -3,44 +3,44 @@
     <div>
       <h5 class="mb-0">
           <a target="_blank" rel="noopener noreferrer"
-             href="https://github.com/dockstore/dockstore/releases/tag/1.14.0">
-              1.14 release</a>
+             href="https://github.com/dockstore/dockstore/releases/tag/1.15.0">
```
Comment (coverbeck) [inline news.html:6]: Tag doesn't exist, but will

Comment (svonworl) [reply news.html:6]: Looks like in the past, we've updated by replacing the previous release announcement with the new release announcement, so this PR conforms.  Which is fine, except it has the effect of eliminating the one existing news item from 2023, making us appear less active than we actually are...

Comment (denis-yuen) [reply news.html:6]: Yeah, maybe dropping the oldest makes more sense

**Review by denis-yuen — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]
(with Steve and Denis' suggestion)

File: news.html (hunk 2 above)
Comment (coverbeck) [reply news.html:6]: Restored 1.14 news, dropping oldest.

I'll wait to merge this until we have a release date.

**Pull request merged**